layouts/layouts.module.css

```
@@ -125,7 +126,7 @@
           lg:max-w-3xl;
 
       > div {
-        @apply w-fit;
+        @apply w-fit max-w-full;
```

```suggestion
        @apply w-fit
          max-w-full;
```

Only one CSS class per line 👀

---

Rgr, surprised formatting didn't catch this?

---

Unfortunately, we do not have a formatter that corrects the class indents